moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.py

```diff
@@ -180,6 +189,16 @@ def create_ikfast_package(args):
     print("Created package.xml at: '%s'" % pkg_xml_path)
 
 
+def find_template_dir():
+  for candidate in [os.path.dirname(__file__) + '/../templates']:
+    if os.path.exists(candidate) and os.path.exists(candidate + '/ikfast.h'):
+      return os.path.realpath(candidate)
+  try:
+    return os.path.join(get_pkg_dir(plugin_gen_pkg), plugin_sub_dir, 'templates')
```

Comment:
If we overwrite get_pkg_dir() with the stub as we did above then this will raise: `AttributeError: 'NoneType' object has no attribute 'endswith'` Am I missing something here?

Reply:
How did you called this? Works for me.
The `get_pkg_dir()` stub raises an exception, which should be catched here. Could you please provide the full python back-trace?

Reply:
My mistake. This will result in the correct behavior.